app.json

```diff
@@ -1,106 +1,129 @@
-        },
-        "MICROMASTERS_FROM_EMAIL": {
-            "description": "E-mail to use for the from field"
-
-        },
-        "NEW_RELIC_APP_NAME": {
-            "description": "Application identifier in New Relic.",
-            "default": "MicroMasters"
-        },
-        "SECRET_KEY": {
-            "description": "Django secret key.",
-            "generator": "secret"
-        },
-        "MICROMASTERS_SECURE_SSL_REDIRECT": {
-            "description": "Application-level SSL redirect setting.",
-            "required": false,
-            "value": "True"
-        },
-        "STATUS_TOKEN": {
-            "description": "Token to access the status API.",
-            "required": true
-        },
-        "EDXORG_BASE_URL": {
-            "description": "The base URL of the edX instance to use for logging in.",
-            "required": true
-        },
-        "EDXORG_CLIENT_ID": {
-            "description": "The OAuth client ID configured in the edX instance.",
-            "required": true
-        },
-        "EDXORG_CLIENT_SECRET": {
-            "description": "The OAuth client secret configured in the edX instance.",
-            "required": true
-        },
-        "AWS_ACCESS_KEY_ID": {
-            "description": "AWS Access Key for S3 storage."
-        },
-        "AWS_SECRET_ACCESS_KEY": {
-            "description": "AWS Secret Key for S3 storage."
-        },
-        "AWS_STORAGE_BUCKET_NAME": {
-            "description": "S3 Bucket name."
-        },
-        "MICROMASTERS_USE_S3": {
-            "description": "Use S3 for storage backend (required on Heroku)",
-            "value": "True"
-        }
+  "addons": [
+    "heroku-postgresql:hobby-dev",
+    "newrelic:wayne"
+  ],
+  "buildpacks": [
+    {
+      "url": "https://github.com/heroku/heroku-buildpack-nodejs"
+    },
+    {
+      "url": "https://github.com/heroku/heroku-buildpack-python"
+    }
+  ],
+  "description": "Web Portal for Micromasters",
+  "env": {
+    "ALLOWED_HOSTS": {
+      "default": "['*']",
+      "description": "Array of allowed hostnames"
+    },
+    "AWS_ACCESS_KEY_ID": {
+      "description": "AWS Access Key for S3 storage."
+    },
+    "AWS_SECRET_ACCESS_KEY": {
+      "description": "AWS Secret Key for S3 storage."
+    },
+    "AWS_STORAGE_BUCKET_NAME": {
+      "description": "S3 Bucket name."
+    },
+    "EDXORG_BASE_URL": {
+      "description": "The base URL of the edX instance to use for logging in.",
+      "required": true
+    },
+    "EDXORG_CLIENT_ID": {
+      "description": "The OAuth client ID configured in the edX instance.",
+      "required": true
+    },
+    "EDXORG_CLIENT_SECRET": {
+      "description": "The OAuth client secret configured in the edX instance.",
+      "required": true
+    },
+    "ELASTICSEARCH_HTTP_AUTH": {
+      "descriptin": "Basic auth settings for connecting to Elasticsearch"
+    },
+    "ELASTICSEARCH_INDEX": {
+      "description": "Index to use on Elasticsearch"
```

Comment:
In that case we should probably remove `./manage.py recreate_index` from the postdeploy so that other deploys don't touch the same instance before we can set this environment variable manually